pipeline/src/transformers/addressables/exhibition.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  asText,
+  asTitle,
+  isFilledLinkToDocumentWithData,
+  isNotUndefined,
+} from '@weco/content-pipeline/src/helpers/type-guards';
+import { ExhibitionPrismicDocument } from '@weco/content-pipeline/src/types/prismic';
+import { ElasticsearchAddressableExhibition } from '@weco/content-pipeline/src/types/transformed';
+
+export const transformAddressableExhibition = (
+  document: ExhibitionPrismicDocument
+): ElasticsearchAddressableExhibition => {
+  const { data, id, uid: documentUid } = document;
+  const format = isFilledLinkToDocumentWithData(data.format)
+    ? asText(data.format.data.title)
+    : 'Exhibition';
+
+  const dates = {
+    start: data.start,
+    end: data.end,
+  };
```

Comment:
For the FE component, should we have the same data name for dates as Events (`times`) or would we treat them differently? Or maybe event should be `dates`, since there's no time 🤔

Reply:
yeah, I wondered about this – I wasn't sure. I guess `dates` is slightly more generic and possibly a better fit even if we include things with times at some point it wouldn't be wrong?